### PR TITLE
Accordion Component

### LINKED ISSUE #321: Accordion Component
Related issue https://github.com/comicrelief/comicrelief-contentful/issues/656

image

```
two children prop

- First child (which is the Title in this case)
- Second child (which is the body in this case)

you click first child and it opens the body
```

### PR DESCRIPTION
**Pull request by LouiaR**
Fixes #321 

https://deploy-preview-327--cr-component-library.netlify.app/#!/Accordion/1

### CODE REVIEW

**Comment by gusliedke** [comment]
Title should be a node as well, so it can have different formats.

File: src/components/Molecules/Accordion/Accordion.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+const Icon = styled.div`
+  display: flex;
+  justify-content: center;
+  align-content: center;
+`;
+
+const Copy = styled.div`
+  overflow: hidden;
+  visibility: none;
+  transition: all 0.2s linear;
```
Comment (AndyEPhipps) [inline src/components/Molecules/Accordion/Accordion.js:57]: Silly, but  howsabout a nicer animation here? Looks a bit stiff using `linear`.. 'pull-up window blind' style?

`transition: all 0.2s cubic-bezier(0.21, 1.7, 0.83, 0.68) 0s;`

**Review by AndyEPhipps — APPROVED** [review]

File: src/components/Molecules/Accordion/Accordion.js (hunk 1 above)
Comment (gusliedke) [reply src/components/Molecules/Accordion/Accordion.js:57]: like it 👍

Comment (AndyEPhipps) [reply src/components/Molecules/Accordion/Accordion.js]: 🎉

**Pull request merged**